Containers/nextcloud/entrypoint.sh

```diff
@@ -548,9 +548,17 @@ if version_greater "$installed_version" "24.0.0.0"; then
     if [ "$IMAGINARY_ENABLED" = 'yes' ]; then
         php /var/www/html/occ config:system:set enabledPreviewProviders 0 --value="OC\\Preview\\Imaginary"
         php /var/www/html/occ config:system:set preview_imaginary_url --value="http://$IMAGINARY_HOST:9000"
+        php /var/www/html/occ config:system:set enabledPreviewProviders 20 --value="OC\\Preview\\HEIC"
+        php /var/www/html/occ config:system:set enabledPreviewProviders 21 --value="OC\\Preview\\SVG"
+        php /var/www/html/occ config:system:set enabledPreviewProviders 22 --value="OC\\Preview\\TIFF"
```

Comment:
Needs https://github.com/nextcloud/server/pull/36352 which hopefully comes with nc25.0.4 but not sure yet